pkg/config/env/olympus.go

```diff
@@ -1,8 +1,30 @@
 package env
 
-import "github.com/gobuffalo/envy"
+import (
+	"runtime"
+	"strconv"
+
+	"github.com/gobuffalo/envy"
+)
 
 // OlympusGlobalEndpointWithDefault returns Olympus global endpoint defined by OLYMPUS_GLOBAL_ENDPOINT.
 func OlympusGlobalEndpointWithDefault(value string) string {
 	return envy.Get("OLYMPUS_GLOBAL_ENDPOINT", value)
 }
+
+// AthensMaxConcurrency retrieves maximal level of concurrency based on ATHENS_MAX_CONCURRENCY.
+// Defaults to number of cores if env is not set.
+func AthensMaxConcurrency() int {
+	defaultMaxConcurrency := runtime.NumCPU()
```

Comment:
if maxConcurrency is `goroutines` can't be like 100 times the number of cpus? Just curious :)

Reply:
it can, but from my experience more is not better, and this is most of the time the sweet spot. 
but we can override in any case